content/blog/2022/platform-tools.md

```diff
@@ -0,0 +1,497 @@
+Many graphic design tools are a step removed
+from the web platform.
+While some do try to export HTML/CSS,
+only a few consider that
+the core goal of the tool.
+Even then, there's been a long-term disconnect
+between a web that's responsive,
+and our almost completely-static design tools.
+
+A few tools have started to provide
+_flexbox-like_ 'responsive' layouts,
+but I haven't seen anything _CSS-grid-like_.
+Most don't even have
+basic support for relative or fluid units.
+Rachel notes in the Almanac
+that most lengths on the web
+are defined in `px`.
+Is that because authors intentionally avoid `em`/`rem`,
+or because the mockups that we receive from designers
+are all limited to `px` by our design tools?
+
+I've [commented on this before](https://twitter.com/TerribleMia/status/1446262147772682260):
+
+> The least responsive part
+> of Responsive Web Design
+> is the design tools.
+> This needs to change.
+
+When it comes to color spaces/gamuts,
+design tools vary in support.
+Looking at some of the big players:
+
+- **Adobe XD**
+  supports HEX, RGB, and HSB.
+  **Illustrator** adds CMYK and a grayscale option,
+  while **Photoshop** also supports extended gamuts
+  such as Display P3,
+  and perceptually-uniform LAB.
+- **Figma**
+  (now also owned by Adobe)
+  provides HEX, RGB,
+  'CSS' (which is `rgba()`),
+  HSL, and HSB.
+- **Sketch**
+  shows HSB, HEX, and RGB(a).
+
+I don't know of any popular design tools
+that provide LCH,
+or the newer okLAB/okLCH variants,
+let alone _relative_ colors
+defined as a function of mixing or adjusting
+other underlying colors.
+
+And when it comes to copying our colors
+out of a design tool,
+the options are generally 6-digit HEX,
+or copying individual channel values,
+one at a time.
+Either way,
+converting our colors
+into a more human-friendly syntax
+would often be an extra step,
+and may not be worth the effort.
+
+Since all legacy colors on the web
+rely on a single color model (RGB)
+and gamut (sRGB),
+there has been little _meaningful_ difference
+between formats (until now).
+Once we used a graphic interface
+to pick the color we want,
+it maybe didn't matter
+what format we copy/paste into
+a variable somewhere.
+
+But what's the designer path
+towards using new, extended color spaces in CSS?
+Can they ever gain traction on the web
+without first finding support in our design tools?
+I'm not sure.
+
+## How CSS language tools intervene
+
+Code tools also play a role in this.
+
+Sass, for example,
+relies on the same
+_interchangeability_ of legacy web formats.
+No matter what syntax an author chooses,
+Sass will store the value as RGB,
+and then allow manipulation
+in any other CSS format.
+But at the end of the day,
+_CSS output is always generated
+in whatever format has the widest browser support_.
+
+It doesn't matter to Sass
+what syntax authors use
+for their _input value_,
+the _output value_ is always
+either HEX or a color-name when opaque,
```

Comment:
```suggestion
either HEX or a color name when opaque,
```
possibly an example of a color name in parentheses if what you mean is something like `rebecaapurple`? Not a biggie.